irohad/network/impl/block_loader_impl.hpp

```diff
@@ -65,6 +70,8 @@ namespace iroha {
       std::shared_ptr<ametsuchi::PeerQuery> peer_query_;
       std::shared_ptr<ametsuchi::BlockQuery> block_query_;
       std::shared_ptr<model::ModelCryptoProvider> crypto_provider_;
+      std::shared_ptr<shared_model::validation::BlockValidator>
+          stateless_validator_;
```

Comment:
Why did you add block stateless validation in this PR? how is it relevant to wrapper?

Reply:
```
The task is to rework poly_wrapper with the shared pointer in block_loader interface. Also, need to remove `libs/common/wrapper.hpp` because the file is redundant.

Also, has to add stateless validation of blocks in the transport layer.
```